Documentation/network/egress-gateway.rst

```diff
@@ -427,3 +427,25 @@ selected Egress IP rather than the one of the node where the pod is running:
     $ tail /var/log/nginx/access.log
     [...]
     192.168.60.100 - - [04/Apr/2021:22:06:57 +0000] "GET / HTTP/1.1" 200 612 "-" "curl/7.52.1"
+
+Troubleshooting
+---------------
+
+To troubleshoot a policy that is not behaving as expected, you can view the
+egress configuration in a cilium agent (the configuration is propagated to all agents,
+so it shouldn't matter which one you pick). 
+
+.. code-block:: shell-session
+
+    $ kubectl -n kube-system exec -it cilium-7r48z -- cilium bpf egress list
+    Defaulted container "cilium-agent" out of: cilium-agent, config (init), mount-cgroup (init), apply-sysctl-overwrites (init), mount-bpf-fs (init), wait-for-node-init (init), clean-cilium-state (init)
+    Source IP    Destination CIDR    Egress IP   Gateway IP
+    192.168.2.23 192.168.60.13/32    0.0.0.0     192.168.60.12
```

Comment:
According to my local testing, this should be:
```suggestion
Source IP    Destination CIDR   Egress IP        Gateway IP
10.72.0.54   192.168.60.13/32   192.168.60.100   10.164.0.67
```
Source IP and Gateway IP depend on IPAM of course, but Egress IP should be 192.168.60.100 AFAICT.

Reply:
Aah, it's 0.0.0.0 on all nodes except the gateway node! Not sure what's the best way to not confuse users with that :thinking: Maybe we have to explain?

Reply:
Interesting! I think it makes sense, have attempted to clarify

Reply:
LGTM.